readme.txt

```diff
@@ -97,10 +97,13 @@ Release and roadmap notes are available on the [WooCommerce Developers Blog](htt
 - Fix: WCPay not working in local payments task #7151
 - Fix: WordPress 5.8 compatibility UI fixes #7255
 - Fix: CurrencyFactory constructor to use proper function #7261
+- Fix: Load Analytics API only when feature is turned on #7193
 - Fix: Currency display on Orders activity card on homescreen #7181
 - Fix: Report export filtering bug. #7165
 - Fix:  Use tab char for the CSV injection prevention. #7154
 - Tweak: Revert Card component removal #7167
+- Tweak: Repurpose disable wc-admin filter to remove optional features #7232
+- Tweak: Revert Card component removal #7167
```

Comment:
This changelog is already present and can be removed, same goes for the change on line 100.